src/cls/async-hooks.ts

```diff
@@ -14,15 +14,15 @@
  * limitations under the License.
  */
 
-// This file requires continuation-local-storage in the AsyncHooksCLS
-// constructor, rather than upon module load.
-import * as asyncHooksModule from 'async_hooks';
+// This file requires async_hooks in the AsyncHooksCLS constructor, rather than
+// upon module load.
```

Comment:
Can you also comment that this import is only for types, and the actual `require` happens in the constructor?

Reply:
It sort of already says that. I think I worded it this way because referencing "this import" doesn't make sense for someone that is looking at the JS file itself, because the import won't be there.

Reply:
Ah, now I get that. Backquoting `require` would've helped me catch that better :)

Reply:
I would prefer if the import symbol was called `asyncHooksTypes` instead of `asyncHooks`. 

```
// This import is for types only. The actual `require` happens in the `AsyncHooksCLS` constructor.
// Do not use `asyncHooksTypes` directly.
```

Reply:
I'll revert it to `asyncHooksModule` as it was before. I don't think `asyncHooksTypes` is accurate because the type of the module is actually `typeof asyncHooksTypes` which doesn't read well.

Our CI should catch `asyncHooksTypes` being used directly because it'll crash in Node 6. I would prefer not to introduce comments that don't make sense in the transpiled code.

Reply:
@jinwoo I've updated it to say `This file calls require('async_hooks') ...` hopefully that should be more clear.